assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -14,15 +14,6 @@
         class="chat-full-page-header__title"
       }}
         {{chat-channel-title channel=chatChannel}}
-
-        {{#if displayMembers}}
```

Comment:
Raised this internally, not sure if we should be removing this for the full screen channel header just yet.

Reply:
If we do then `displayMembers` computed property in the JS must be removed as well.